CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+### 23.2.0 [#681](https://github.com/openfisca/openfisca-core/pull/681)
+
+* Change the way metadata are declared for Parameter.
+
+Before:
+```YAML
+description: Age of retirement
+reference: https://wikipedia.org/wiki/retirement
```

Comment:
The idea is to do a minimal change to the actual situation.
And description can be thought as standard/generic and informative enough
to be placed outside metadata which is more for custom/non-vital metadata.
But @fpagnoux may have more details

Reply:
What would such an example illustrate? 

Right now the metadata are just a pass-trough, and core has no opinion about the content of this `reference`, so I'm not sure about the value of making the changelog more complex by adding an other slightly different example.

Reply:
I was asking because it was the first thing that came into my mind when I saw the singular `reference` and that there was only one reference :)
It might just be a worry I have :)

Reply:
Yep, we should keep that in mind.